homebridge/config.schema.json

```diff
@@ -8,13 +8,18 @@
       "email": {
         "title": "Email",
         "type": "string",
+        "required": true,
```

Comment:
Unfortunately we can't make email/password required.  The way I have everything set up, you can use _either_ `email` + `password` _or_ `refreshToken`.  The json schema form generator doesn't appear to have a mechanism for supporting this (if it does and I overlooked it, please let me know).  I have `oneOf` clause at the bottom that sound properly require one or the other, but it seems to be ignored.